utils/cli.py

```diff
@@ -49,7 +53,13 @@ def args_validation(args) -> bool:
     if args.llama:
         from transformers import LlamaForCausalLM, LlamaTokenizer
 
+    if args.vllm:
+        from vllm import AsyncEngineArgs, AsyncLLMEngine, SamplingParams
+
     if args.trust_remote_code is False and args.model_version in "0.5 0.7 0.8 0.9":
         raise ValueError("If you use model version 0.5, 0.7, 0.8 or 0.9, please add flag --trust_remote_code.")
 
-    return True
\ No newline at end of file
+    if args.use_gptq_model and args.use_awq_model:
```

Comment:
I think these checkings should be moved into cli.py `args_validation` function? or we just deprecate the cli.py::args_validation and keep all checking in `load_model`?
@pipixia244

Reply:
In my opinion the `args_validation` should only check whether the args are valid (such as no both `use_gptq_model` and `llama_cpp`), and the module importing part should be in `load_model`.